raspberrypi-c.js

```diff
@@ -21,6 +23,18 @@ var COMPILER_FOLDER = all.getToolsFolder() + '/' + COMPILER_NAME + '/bin';
 //  all.gulpTaskBI(gulp, 'c', 'RaspberryPi', 'az-blink');
 //}
 
+// XXX - this function shall be replaced
+function azhRunLocalCmd(cmd, verbose, cb) {
```

Comment:
yes, that was my plan, to discuss raspberry pi node changes with you and then change along with raspberry pi changes